app/models/subscription.rb

```diff
@@ -58,7 +58,9 @@ def subscribe!(brand, subscription_plan, email, stripe_token)
       if customer
         stripe_subscription_id = customer.subscriptions.first.id
         create_payment_handler!(brand, customer)
-        create_subscription!(brand, subscription_plan, stripe_subscription_id, trial_end)
+        subscription = create_subscription!(brand, subscription_plan, stripe_subscription_id, trial_end)
+        UserMailer.welcome(brand)
```

Comment:
That third argument I'd actually do like

```ruby
module TransactionalMailers
  def welcome(brand)
    self.new(
      self::Types::WELCOME,
      brand,
      {
        FOO_BAR: 'I am the content for foobar'
      },
    )
  end
end
```

and then have a translation with `TransactionalEmail` that converts the hash to an array of hashes

```ruby
class TransactionalEmail
  def initialize(template_name, brand, merge_vars)
    # ...
    @merge_vars = merge_vars
  end

  def mandrill_merge_vars
    merge_vars.map do |name, content|
      { name: name, content: content }
    end
  end
end
```